src/solidity-frontend/solidity_convert.h

```diff
@@ -209,7 +210,9 @@ class solidity_convertert
   contextt &context;
   namespacet ns;
   // json for Solidity AST. Use vector for multiple contracts
-  nlohmann::json &src_ast_json;
+  nlohmann::json src_ast_json_array = nlohmann::json::array();
```

Comment:
Do we need this default value? I thought it will be initialized in the constructor initialize list.

Reply:
I've used the same display construct here as in language to distinguish it from the single AST structure below, which I think makes the code easier to read.